src/Components/SmartComponents/CVEs/CVEsTableToolbar.js

```diff
@@ -39,6 +39,7 @@ const CVEsTableToolbarWithContext = (props) => {
     };
 
     const { context } = props;
+    const { canEditStatusOrBusinessRisk, canExport } = props;
```

Comment:
It's common practice to destructure all variables you need in one statement:
```
    const { context } = props;
    const { canEditStatusOrBusinessRisk, canExport } = props;
```
would be better as
```
    const { context, canEditStatusOrBusinessRisk, canExport } = props;
```